contrib/capabilities-tester/Makefile

```diff
@@ -1,4 +1,4 @@
 GCC ?= gcc
 
 test_caps: test_caps.c
-	$(GCC) -Wall $< -o $@ -static -lcap
+	$(GCC) -Wall $< -o $@ -lcap
```

Comment:
Out of curiosity, why this breaks build on RHEL8?

Reply:
did you look at the references in the issue description?